src/libraries/System.Drawing.Primitives/tests/ColorTests.cs

```diff
@@ -130,6 +130,7 @@ public class ColorTests
                 new object[] {"Plum", 255, 221, 160, 221},
                 new object[] {"PowderBlue", 255, 176, 224, 230},
                 new object[] {"Purple", 255, 128, 0, 128},
+                new object[] {"RebeccaPurple", 255, 102, 51, 153},
```

Comment:
You should check if the tests have the same requirement as well.

Reply:
will check to see if unit tests will have the requirement soon

Reply:
i was unable to check tests today because of visual studio error :/
but i will do it hopefully tomorow.